### ISSUE TITLE
Global variables leaking from hsluv module

### ISSUE THREAD

**Issue by bluss** [yorik1984/newpaper.nvim#6]
Hi, it looks like the hsluv module is exporting a bunch of should-be-local variables into the global namespace. These end up in the global namespace for everyone loading the plugin.

See code: https://github.com/yorik1984/newpaper.nvim/blob/0fff7080666aa41aa0e0b4e32db9788c396da479/lua/newpaper/hsluv.lua#L23-L34

Common mistake in lua that happens easily I guess.

To make them local would make them encapsulated, and you avoid risk - people using the unintentionally used variables, and people unintentionally overriding these variables since they are global.

Lua language server says

```
 │     Global variable in lowercase initial, Did you miss `local` or misspell it? Lua Diagnostics. (lowercase-global) [23, 1]
 │     Global variable in lowercase initial, Did you miss `local` or misspell it? Lua Diagnostics. (lowercase-global) [25, 1]
 │     Global variable in lowercase initial, Did you miss `local` or misspell it? Lua Diagnostics. (lowercase-global) [28, 1]
 │     Global variable in lowercase initial, Did you miss `local` or misspell it? Lua Diagnostics. (lowercase-global) [32, 1]
```

**Comment by yorik1984** [comment]
fix in 855067b5897c58ad57fdceaa7731853a94692bc4

**Comment by yorik1984** [comment]
fix 855067b5897c58ad57fdceaa7731853a94692bc4

**Comment by bluss** [comment]
thanks
